package-lock.json

```diff
@@ -14,10 +14,49 @@
       },
       "devDependencies": {
         "@parcel/transformer-sass": "^2.8.0",
+        "babel-loader": "^9.1.2",
+        "clean-webpack-plugin": "^4.0.0",
         "concurrently": "^7.6.0",
+        "copy-webpack-plugin": "^11.0.0",
+        "css-loader": "^6.7.3",
+        "html-loader": "^4.2.0",
+        "html-webpack-plugin": "^5.5.0",
+        "mini-css-extract-plugin": "^2.7.2",
         "parcel": "^2.8.0",
```

Comment:
**Nit (clean code):** We should uninstall parcelJS if we're not using it anymore.

Reply:
Good call, I had thought I removed it! Thanks for double checking